src/rpc/RPCHelpers.cpp

```diff
@@ -411,18 +521,37 @@ traverseOwnedNodes(
     BackendInterface const& backend,
     ripple::AccountID const& accountID,
     std::uint32_t sequence,
-    ripple::uint256 const& cursor,
+    ripple::uint256 const& marker,
     boost::asio::yield_context& yield,
     std::function<bool(ripple::SLE)> atOwnedNode)
 {
     if (!backend.fetchLedgerObject(
             ripple::keylet::account(accountID).key, sequence, yield))
         throw AccountNotFoundError(ripple::toBase58(accountID));
-    auto const rootIndex = ripple::keylet::ownerDir(accountID);
+
+    return traverseOwnedNodes(
+        backend,
+        ripple::keylet::ownerDir(accountID),
+        sequence,
+        marker,
+        yield,
+        atOwnedNode);
+}
+
+std::optional<ripple::uint256>
```

Comment:
I originally wrote this function in a very naive way. In this PR, it pulls every object each time you query. So if you're paging through `n` times, you actually read the first owned node `n` times. This is _really_ stupid, and totally on me. :facepalm: 

I modified `traverseOwnedNodes` to take a hint in the `cursor` such that we didn't have to traverse from the first directory every time.

Can we rebase?

Reply:
Sure. Will rebase before merging.

Reply:
@natenichols please double check to make sure I'm using the modified function correctly.